packages/core-types/src/ChainsTypes.ts

```diff
@@ -28,5 +30,10 @@ export enum ChainId {
   ASTAR_TESTNET = 81,
   CHILIZ_MAINNET = 88888,
   CHILIZ_TESTNET = 88882,
+  CORE_MAINNET = 1116,
```

Comment:
Are these already added in ChainId in core-types?

Reply:
This is the chainId enum in core-types package